frontend/src/app/__snapshots__/app.test.tsx.snap

```diff
@@ -85,6 +85,20 @@ exports[`App tests should render default App component 1`] = `
               className=""
               dataListCells={
                 Array [
+                  <DataListCell
+                    alignRight={false}
+                    className=""
+                    isFilled={true}
+                    isIcon={false}
+                    width={1}
```

Comment:
I'm curious about this name={value} syntax. I haven't seen it anywhere else before, but I think I have only worked with name="stringvalue" so far. Are the brackets necessary for non-string types?

Reply:
This seems to have been generated automatically by the tests. I have not made the changes.